back/prisma/schema.prisma

```diff
@@ -145,17 +145,18 @@ model profile_pictures {
 }
 
 model conversation {
-  id                                              Int       @id @default(autoincrement())
-  creator_id                                      Int
-  member_id                                       Int
-  creation_date                                   DateTime  @db.DateTime(0)
-  latest_message_id                               Int?
-  update_date                                     DateTime  @db.DateTime(0)
-  account_conversation_creator_idToaccount        account   @relation("conversation_creator_idToaccount", fields: [creator_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "conversation_ibfk_1")
-  account_conversation_member_idToaccount         account   @relation("conversation_member_idToaccount", fields: [member_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "conversation_ibfk_2")
-  message_conversation_latest_message_idTomessage message?  @relation("conversation_latest_message_idTomessage", fields: [latest_message_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "conversation_ibfk_3")
-  message_message_conversation_idToconversation   message[] @relation("message_conversation_idToconversation")
-
+  id                Int       @id @default(autoincrement())
+  creator_id        Int
+  member_id         Int
+  creation_date     DateTime? @db.DateTime(0) @default(now())
+  latest_message_id Int?
```

Comment:
I thought that we didn't add the lastest message in the conversation, did we change ?

Reply:
I'll answer IRL

Reply:
Soooo.... it's a date ?

Reply:
The fruit ?